### ISSUE TITLE
Only Darkmode seems supported caused by including app.css

### ISSUE THREAD

**Issue by tonypartridger** [heloufir/filament-workflow-manager#15]
When initiating the package my light theme is destroyed. Upon removing the app.css call from the Workflow-Manager distribution all seems ok again. I am guessing there are CSS classes/variables within the package which shouldn't exist and should be namespaced to the package alone?

![Screenshot 2023-02-07 at 13 35 39](https://user-images.githubusercontent.com/103045669/217259483-38535f89-3c57-4da1-a8df-37448d8117f7.png)
![Screenshot 2023-02-07 at 13 35 46](https://user-images.githubusercontent.com/103045669/217259492-cc8b7264-6cb3-4b48-9607-4709e786b078.png)

**Comment by sergioraldana** [comment]
Same here

**Comment by kaiserification** [comment]
Same here

**Comment by tonypartridge** [comment]
Yep, it looks to be abandoned due to lack of communication / updates?
